crates/networking/rpc/rpc.rs

```diff
@@ -56,6 +62,7 @@ pub struct RpcApiContext {
     storage: Store,
     jwt_secret: Bytes,
     local_p2p_node: Node,
+    active_filters: ActiveFilters,
```

Comment:
this is good for now, but maybe eventually we need to have a different approach than having a random list of things that some endpoints might need. but def out of scope of this PR.

Reply:
I agree, we could leverage Axum's tools for this, but maybe on another issue/PR.